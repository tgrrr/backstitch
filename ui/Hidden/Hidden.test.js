import React from 'react'
import renderer from 'react-test-renderer'
import { Hidden } from '.'

it('Hidden: default', () => {
    const component = renderer.create(
        <Hidden>
            <>
                Hidden
            </>
        </Hidden>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
