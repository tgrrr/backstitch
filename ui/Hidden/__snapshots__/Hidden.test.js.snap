// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hidden: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Hidden "
    data-testid="Hidden"
  />
</div>
`;
